deployment/src/strongmind_deployment/acm.py

```diff
@@ -0,0 +1,60 @@
+import re
+from dataclasses import dataclass
+from pulumi import Output, ResourceOptions
+from pulumi_aws import acm, route53
+import pulumi
+
+
+@dataclass
```

Comment:
this dataclass has introduced issues.  I don't think Mark was a big fan of it either, so we can remove that.  

Pydantic is an option but might need to test it out a bit first.  I had some issues with pydantic.

Reply:
I'm actually good with dataclasses, pydantic is like dataclasses++, as it has validations (which might be really good for making sure that teams don't send non-useful elements to the components, and improving their user experience when they do so).

Reply:
replaced with pydantic